jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -1198,6 +1199,11 @@ YUI.add('juju-env-api', function(Y) {
         'num-units': numUnits
       };
 
+      if (series) {
+        // We only want to supply the series when it is a multi-series charm.
```

Comment:
Why? If series are passed I think we want to always provide it, and your code actually does that.